client/termdb/TermdbVocab.js

```diff
@@ -527,7 +535,7 @@ export class TermdbVocab extends Vocab {
 			if (filter) {
 				body.filter = getNormalRoot(filter)
 			}
-			return await dofetch3('/termdb', { body })
+			return await dofetch3('termdb/categories', { headers, body })
```

Comment:
this breaks [snplst term](http://localhost:3000/?noheader=1&mass={%22genome%22:%22hg38%22,%22dslabel%22:%22SJLife%22,%22nav%22:{%22activeTab%22:1},%22plots%22:[{%22chartType%22:%22regression%22,%22regressionType%22:%22linear%22,%22outcome%22:{%22id%22:%22LV_Ejection_Fraction_3D%22},%22independent%22:[{%22term%22:{%22type%22:%22snplst%22,%22snps%22:[{%22rsid%22:%22rs1333261826%22,%22effectAllele%22:%22T%22},{%22rsid%22:%22rs1010330999%22,%22effectAllele%22:%22G%22},{%22rsid%22:%22rs1641548%22,%22effectAllele%22:%22T%22},{%22rsid%22:%22rs858528%22,%22effectAllele%22:%22T%22},{%22rsid%22:%22rs1642793%22,%22effectAllele%22:%22C%22},{%22rsid%22:%22rs1042522%22},{%22rsid%22:%22rs1642782%22},{%22rsid%22:%22rs6503048%22},{%22rsid%22:%22rs1472752888%22,%22effectAllele%22:%22T%22}]},%22q%22:{%22AFcutoff%22:5,%22alleleType%22:0,%22geneticModel%22:0,%22missingGenotype%22:0,%22restrictAncestry%22:{%22name%22:%22European%20ancestry%22,%22tvs%22:{%22term%22:{%22id%22:%22genetic_race%22,%22type%22:%22categorical%22,%22name%22:%22Genetically%20defined%20race%22},%22values%22:[{%22key%22:%22European%20Ancestry%22,%22label%22:%22European%20Ancestry%22}]}}}},{%22id%22:%22sex%22,%22refGrp%22:%222%22}]}]}), please revert. see my previous commit

Reply:
okay. Let me see. For now, I have converted it to a draft.